### PR TITLE
chore: update cosmosnetwork discord url

### PR DESCRIPTION
**Pull request by polymaer**
I updated all the old discord url to https://discord.gg/interchain or https://discord.com/invite/interchain, since the vanity link for the Cosmos discord has changed to interchain.

### CODE REVIEW

**Review by melekes — APPROVED** [review]
👍

**Comment by polymaer** [comment]
@melekes Hi sir, can i get this PR merged? 😄

File: .changelog/v0.34.27/summary.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ our [upgrading guidelines](./UPGRADING.md).
 If you have any questions, comments, concerns or feedback on this release, we
 would love to hear from you! Please contact us via [GitHub
 Discussions](https://github.com/cometbft/cometbft/discussions),
-[Discord](https://discord.gg/cosmosnetwork) (in the `#cometbft` channel) or
+[Discord](https://discord.gg/interchain) (in the `#cometbft` channel) or
```
Comment (melekes) [inline .changelog/v0.34.27/summary.md:13]:
```suggestion
[Discord](https://discord.gg/cosmosnetwork) (in the `#cometbft` channel) or
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1093,7 +1093,7 @@ our [upgrading guidelines](./UPGRADING.md).
 If you have any questions, comments, concerns or feedback on this release, we
 would love to hear from you! Please contact us via [GitHub
 Discussions](https://github.com/cometbft/cometbft/discussions),
-[Discord](https://discord.gg/cosmosnetwork) (in the `#cometbft` channel) or
+[Discord](https://discord.gg/interchain) (in the `#cometbft` channel) or
```
Comment (melekes) [inline CHANGELOG.md:1096]:
```suggestion
[Discord](https://discord.gg/cosmosnetwork) (in the `#cometbft` channel) or
```

**Comment by melekes** [comment]
@mergify backport v0.38.x

**Pull request merged**